frontend/src/components/MeetingRoom/VideoTile/VideoTile.tsx

```diff
@@ -50,6 +50,9 @@ const VideoTile = forwardRef(
         <div
           className={`relative left-0 top-0 size-full overflow-hidden rounded-xl ${isTalking ? 'outline outline-2 outline-sky-500' : ''} ${!hasVideo ? 'hidden' : ''}`}
           ref={ref}
+          style={{
+            backgroundColor: 'rgba(60, 64, 67, 0.55)',
```

Comment:
how come we're using style for everything in this PR?

Reply:
I changed most of them back. This one needs to stay, though.

Using `style` made it easier to test different props with the devTools. If I manually edited it through the `.object-contain`, `object-fill`, etc., classes, it would make changes for _all_ of the elements with that class.